CHANGELOG.md

```diff
@@ -16,8 +16,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 ### Features
 ### Enhancements
 ### Bug Fixes
-- Fix multi node "no such index" error in text chunking processor ([#713](https://github.com/opensearch-project/neural-search/pull/713))
 ### Infrastructure
 ### Documentation
+- Include latest change in release notes ([#734](https://github.com/opensearch-project/neural-search/pull/734))
```

Comment:
Lets remove this - I dont get much information from the note. I think it can be skipped.

Reply:
Sure. Will removing this cause change log failure in github checks?